logstash/README.md

```diff
@@ -31,9 +32,12 @@ When you run `datadog-agent info` you should see something like the following:
 
 The logstash check is compatible with Logstash 5.6 and possible earlier versions. Currently it does not support the new pipelines metrics in Logstash 6.0 yet.
 
-## Metrics
+## Data Collected
+### Metrics
+See [metadata.csv](https://github.com/DataDog/integrations-extras/blob/master/filebeat/metadata.csv) for a list of metrics provided by this integration.
 
-See [metadata.csv](metadata.csv) for a list of metrics provided by this integration.
+### Events
+The Filebeat check does not include any event at this time.
```

Comment:
The Logstash integration doc mentions/links to the filebeat one multiple times.

Reply:
👌